packages/mdl-theme/_variables.scss

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "./functions";
+
+/*
+  Main theme colors.
+  If you're a user customizing your color scheme in SASS, these are probably the only variables you need to change.
+*/
+$mdl-theme-primary: #3f51b5 !default; /* Indigo 500 */
+$mdl-theme-accent: #ff4081 !default; /* Pink A200 */
+$mdl-theme-background: #fff !default; /* White */
+
+/* Which set of text colors to use for each main theme color (light or dark) */
+$mdl-theme-text-on-primary: mdl-theme-light-or-dark($mdl-theme-primary);
```

Comment:
+1 to tone naming. It does make more sense here (albeit I do have extremely limited color understanding.)